app_dart/lib/src/datastore/cocoon_config.dart

```diff
@@ -77,6 +80,17 @@ class Config {
     return Uint8List.fromList(result.value.codeUnits);
   }
 
+  // GitHub App properties.
+  Future<String> get githubPrivateKey =>
+      _getSingleValue('githubapp_private_pem');
+  Future<String> get githubPublicKey => _getSingleValue('githubapp_public_pem');
+  Future<String> get githubAppId => _getSingleValue('githubapp_id');
```

Comment:
curious how do we obtain these keys?

Reply:
These keys are generated from the github application configuration page. We only need a set of keys which the application will use to sign short lived tokens to talk to the apis.